src/cmd/run.go

```diff
@@ -120,25 +120,20 @@ func run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if err := runCommand(container,
+	return runCommand(container,
 		!nonDefaultContainer,
 		image,
 		release,
 		command,
 		false,
-		false,
-		true); err != nil {
-		return err
-	}
-
-	return nil
+		true)
 }
 
 func runCommand(container string,
 	defaultContainer bool,
 	image, release string,
 	command []string,
-	emitEscapeSequence, fallbackToBash, pedantic bool) error {
```

Comment:
I removed fallbackToBash as it was only used in the enter command since the shell can now be specified. I'm not sure if I should have kept it or not though.